### DISCUSSION TITLE
Bypass mode

### DISCUSSION THREAD

**Discussion by danyacosta91** [jblance/mpp-solar#249] [Q&A]
Hi,

I have a MPP solar inverter and using the mpp-solar on a raspberry PI 4 with the `mpp-solar -p /dev/hidraw0 --getstatus -o json` command to get the data.

How can I know if the inverter is in bypass mode from this command? or is there a way to get that information from the inverter?

**Comment by meccip** [comment]
I have easun sv4. this work for me
mpp-solar -p /dev/hidraw0 -P PI18 -c MOD
